### PR TITLE
Rename "private" file, doc doc changes in CHANGES

### PR DESCRIPTION
**Pull request by richsalz**
This also fixes a long-standing source of confusion, dating dating back to when I first created the file.  "private.num" is really "defines.num"

Removed the -p option; it's always on now.

### CODE REVIEW

**Comment by levitte** [comment]
"defines" is also not quite right, or `.num` for that matter.  `extraneous.txt`, perhaps?

**Comment by richsalz** [comment]
Happy to change the name to *anything* more useful just tell me what it should be.  The current name was a bad mistake on my part.

File: CHANGES [hunk 1]
```diff
@@ -9,6 +9,11 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Hundreds of improvements were made to the documentation, including:
+     adding missing command flags, better style conformance, documentation
+     of internals, etc.
+     [Many]
```
Comment (mspncp) [inline CHANGES]: „Hundreds of improvements...“ by „[Many]“?  Sounds a little bit exaggerated and vague at the same time. 

I‘d suggest:

„A lot of improvements...“ by „[Rich Salz and others]“. (or „e.a.“)

Comment (mspncp) [reply CHANGES]: If you think that others made outstanding contributions to the documentation and should be mentioned too, feel free to add them.

**Comment by mspncp** [comment]
> "defines" is also not quite right, or .num for that matter. extraneous.txt, perhaps?

How about „extra.sym“ or „extra.syms“?

File: CHANGES (hunk 1 above)
Comment (richsalz) [reply CHANGES]: I made\ over 2000 changes via ```git diff master | grep '^+' | wc -l``` so I reworded it and put myself as the author.  Other folks should add themselves.

**Comment by richsalz** [comment]
"missing.num" is now called "other-names.num"

**Comment by levitte** [comment]
Frankly, .num is wrong.

File: CHANGES (hunk 1 above)
Comment (mspncp) [reply CHANGES]: Oh, I see. I wasn't aware that the "Hundreds of improvements" were measured in line counts ;-)

Comment (mspncp) [reply CHANGES]: > put myself as the author

You truly deserve the credits IMHO. Nobody else put so much effort into cleaning up the documentation as you did recently.

**Comment by mspncp** [comment]
Let's call it `without.num` ;-)

**Comment by levitte** [comment]
I'm serious in my pickiness

**Comment by richsalz** [comment]
@levitte I no longer care, tell me what name you want.

**Comment by levitte** [comment]
`util/other.syms`

**Comment by levitte** [comment]
(I do like the `.syms` extension.  Thanks for that idea, @mspncp)

**Comment by richsalz** [comment]
Not thrilled with the non-common ".syms" extension, but I did it.

**Comment by levitte** [comment]
It's not like ".num" is much more common. :wink:

File: util/find-doc-nits [hunk 2]
```diff
@@ -512,14 +509,15 @@ sub checklinks {
...
+# Load the public symbol/macro names
 sub publicize {
     foreach my $name ( parsenum('util/libcrypto.num') ) {
         $public{$name} = 1;
     }
     foreach my $name ( parsenum('util/libssl.num') ) {
         $public{$name} = 1;
     }
-    foreach my $name ( parsenum('util/private.num') ) {
+    foreach my $name ( parsenum('util/other-names.num') ) {
```
Comment (levitte) [inline util/find-doc-nits]: Uhmm...

Comment (richsalz) [reply util/find-doc-nits]: rats, that's the second time I've forgotten that.  updated commit pushed.

**Comment by richsalz** [comment]
I would wait until the build-fixing PR goes in; I will have to rebase.

**Comment by richsalz** [comment]
Rebased and pushed.  I don't know if this needs another review or not.

**Comment by levitte** [comment]
> I don't know if this needs another review or not.

It was a clean rebase, yeah?  In that case, no

**Comment by richsalz** [comment]
There were merge issues in find-doc-nits, as was to be expected.

**Comment by levitte** [comment]
In that cases, it's better to re-review

**Comment by richsalz** [comment]
Should be just a reconfirm, the changes to find-doc-nits in this PR were simple.

**Comment by levitte** [comment]
Merged.

185ec4be6d3beac8cb262415c48e78b129f7efd2 Rename "private" file, doc doc changes in CHANGES

**Pull request closed without merging**